apps/src/applab/CrosshairOverlay.js

```diff
@@ -116,15 +145,11 @@ CrosshairOverlay.prototype.create_ = function () {
   this.hGuide_.setAttribute('x2', 0);
   this.ownElement_.appendChild(this.hGuide_);
 
-  this.bubble_ = document.createElementNS(SVG_NS, 'rect');
-  this.bubble_.setAttribute('width', TEXT_RECT_WIDTH);
-  this.bubble_.setAttribute('height', TEXT_RECT_HEIGHT);
-  this.bubble_.setAttribute('rx', TEXT_RECT_RADIUS);
-  this.bubble_.setAttribute('ry', TEXT_RECT_RADIUS);
-  this.ownElement_.appendChild(this.bubble_);
+  // Create the cooordinates tooltip
+  [this.bubble_, this.text_] = this.createTextBubbleElement_();
```

Comment:
[Destructuring assignment is part of ES2015](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Specifications) which we're not _technically_ supporting in our code yet (though we're all anxious to get to it).